README.rst

```diff
@@ -59,13 +59,13 @@ These error codes are emitted:
 +------+-------------------------------------------------------+
 | N813 | camelcase imported as lowercase                       |
 +------+-------------------------------------------------------+
-| N814 | camelcase imported as constant                        |
+| N814 | camelcase imported as constant (see also ``N817``)    |
 +------+-------------------------------------------------------+
 | N815 | mixedCase variable in class scope                     |
 +------+-------------------------------------------------------+
 | N816 | mixedCase variable in global scope                    |
 +------+-------------------------------------------------------+
-| N817 | camelcase imported as acronym                         |
+| N817 | camelcase imported as acronym (see also ``N814``)     |
```

Comment:
```suggestion
| N817 | camelcase imported as acronym (see also ``N814``) |
|      | separated from ``N814`` for easy ignoring         |
```

I'm pretty sure that rST allows you to have multiple lines in a table row like this. I think this is all that would satisfy the request in the original issue.